packages/react-codemirror/src/e2e_tests/signatureHelp.spec.tsx

```diff
@@ -103,8 +107,8 @@ test('Signature help shows the description for the first argument when the curso
 
   await testTooltip(tooltip, {
     includes: [
-      'nodes :: LIST<MAP>',
-      'Imports `NODE` and `RELATIONSHIP` values with the given labels and types from the provided CSV file',
+      importCsvProc.argumentDescription[0].description,
+      importCsvProc.description,
```

Comment:
Just makes these tests more mantainable by not having to update strings all the time